packages/volto/cypress/tests/core/blocks/blocks-slate-delete.js

```diff
@@ -0,0 +1,84 @@
+describe('Slate Delete key behavior', () => {
+  beforeEach(() => {
+    cy.intercept('GET', `/**/*?expand*`).as('content');
+    cy.intercept('GET', '/**/Document').as('schema');
+
+    cy.autologin();
+    cy.createContent({
+      contentType: 'Document',
+      contentId: 'my-page',
+      contentTitle: 'My Page',
+    });
+    cy.visit('/my-page');
+    cy.wait('@content');
+    cy.navigate('/my-page/edit');
+    cy.wait('@schema');
+  });
+
+  it('Delete at end of A merges B text block into A and removes B', () => {
+    cy.getSlate().click().typeWithDelay('First block text');
+    cy.getSlate().should('contain', 'First block text');
+
+    cy.addNewBlock('slate');
+    cy.getSlateEditorAndType('Second block text');
+
+    cy.get(
+      '.content-area .block-editor-slate .slate-editor [contenteditable=true]',
+      { timeout: 8000 },
```

Comment:
You can use `cy.getSlateEditorSelectorAndType` See:

https://github.com/plone/volto/blob/835e9f173f4e7eb07ed87925f3de4bad95a50a30/packages/volto/cypress/support/commands.js#L845

In other places too.